config/config.go

```diff
@@ -437,6 +437,13 @@ type OgcAPI3dGeoVolumes struct {
 
 	// Collections to be served as 3D GeoVolumes
 	Collections GeoSpatialCollections `yaml:"collections" json:"collections"`
+
+	// Whether GeoJSON/JSON-FG responses will be validated against the OpenAPI spec
```

Comment:
GeoJSON/JSON-FG -> JSON (het is namelijk 3d tiles of quantized mesh)

Reply:
Ho oeps, vergeten aan te passen na kopiëren